backend/db/migrate/20230608080932_rename_api_entreprise_scopes_in_enrollments.rb

```diff
@@ -0,0 +1,103 @@
+class RenameApiEntrepriseScopesInEnrollments < ActiveRecord::Migration[7.0]
+  def up
+    Enrollment.find_each do |enrollment|
+      scopes = enrollment.scopes.map do |scope|
+        case scope
+        when "associations"
+          "open_data"
+        when "associations_donnees_protegees"
+          "associations_djepva"
+        when "attestations_fiscales"
+          "attestation_fiscale_dgfip"
+        when "attestations_sociales"
+          "attestation_sociale_urssaf"
+        when "bilans_entreprise_bdf"
+          "bilans_bdf"
+        when "fntp_carte_pro"
+          "open_data"
+        when "certificat_cnetp"
+          "certification_cnetp"
+        when "msa_cotisations"
+          "cotisations_msa"
+        when "certificat_opqibi"
+          "certification_opqibi"
+        when "probtp"
+          "cotisations_probtp"
+        when "qualibat"
+          "open_data"
+        when "certificat_rge_ademe"
+          "open_data"
+        when "exercices"
+          "chiffre_affaires_dgfip"
+        when "extrait_court_inpi"
+          "open_data"
+        when "extraits_rcs"
+          "open_data"
+        when "entreprises"
+          "mandataires_sociaux_infogreffe"
+        when "etablissements"
+          "open_data" && "unites_legales_etablissements_insee"
+        when "liasse_fiscale"
+          "liasses_fiscales_dgfip"
+        when "actes_inpi"
+          "open_data"
+        when "bilans_inpi"
+          "comptes_annuels_inpi"
+        when "conventions_collectives"
+          "open_data"
+        when "entreprises_artisanales"
+          "open_data"
+        when "eori_douanes"
+          "open_data"
+        when "certificat_agence_bio"
+          "open_data"
+        else
+          scope
+        end
+      end
+
+      Enrollment.where(target_api: "api_entreprise").update_all(scopes: scopes)
```

Comment:
```suggestion
      enrollement.update(scopes: scopes)
```

Autrement tu mets à jour TOUTE la db à chaque fois avec les scopes d'une demande. In fine on va se retrouver donc avec toutes les demandes DataPass d'API Entreprise avec les scopes de la dernière demande DataPass.